.github/workflows/pull-request.yml

```
@@ -0,0 +1,111 @@
+
+      - name: Cache Xcode derived data
+        uses: actions/cache@v4
+        with:
+          path: ios/build
+          key: ${{ runner.os }}-xcode-derived-data-${{ steps.fingerprint.outputs.hash }}
+          restore-keys: |
+            ${{ runner.os }}-xcode-derived-data-
+
+      - name: Build
+        working-directory: ./ios
+        run: set -o pipefail && xcodebuild build -workspace Lobbium.xcworkspace -scheme Lobbium -configuration Debug -destination "generic/platform=iOS Simulator" -derivedDataPath build | xcbeautify
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)